docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,156 @@
+- *placement_policy_json* represents a JSON string that specifies the placement policy for this tablespace. The JSON structure contains the following two fields: 
+  - `num_replicas` defines the overall replication factor.
+  - `placement_blocks` is an array of tuples, with each tuple containing the keys `<”cloud”, “region”, “zone”, “min_num_replicas”>` whose values define a placement block. Typically, the sum of `min_num_replicas` across all placement blocks is expected to be equal to `num_replicas`. The aggregate of `min_num_replicas` can be lesser than `num_replicas`, in which case the extra replicas are placed at the YB-Load balancer’s discretion.
+
+### How to Create a Multi-Zone Tablespace
+
+The following example shows how to create a multi-zone tablespace:
+
+```sql
+CREATE TABLESPACE us_west_tablespace 
+WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
+[{"cloud":"aws","region":"us-west","zone":"us-west-1a","min_num_replicas":1},
+{"cloud":"aws","region":"us-west","zone":"us-west-1b","min_num_replicas":1},
+{"cloud":"aws","region":"us-west","zone":"us-west-1c","min_num_replicas":1}]}');
+```
+
+If you create a table and use the preceding tablespace, each tablet of the table will have three replicas, as per the `num_replicas` value. Since `placement_blocks` contains three placement blocks where `min_num_replicas` is set to 1, YB-Load balancer will ensure that `aws.us-west.us-west-1a`, `aws.us-west.us-west-1b`, and `aws.us-west.us-west-1c` will each have one replica.
+
+The following example demonstrates how to create a tablespace in which the value of `min_num_replicas` does not correspond to `num_replicas`:
+
+```sql
+CREATE TABLESPACE us_east_tablespace 
+WITH (replica_placement='{"num_replicas": 5, "placement_blocks":
+[{"cloud":"aws","region":"us-east","zone":"us-east-1a","min_num_replicas":1},
+{"cloud":"aws","region":"us-east","zone":"us-east-1b","min_num_replicas":1},
+{"cloud":"aws","region":"us-east","zone":"us-east-1c","min_num_replicas":1}]}');
+```
+
+Even though `us_east_tablespace` shown in the preceding example has been created successfully, a notice is displayed after the execution stating that the additional two replicas are to be placed at the YB-Load balancer’s discretion in  `aws.us-east.us-east-1a`, `aws.us-east.us-east-1b`, and `aws.us-east.us-east-1c`. Based on the load, `aws.us-east.us-east-1a` may have three replicas and the other zones may have one, or it might later change such that `aws.us-east.us-east-1a` and `aws.us-east.us-east-1b` have two replicas each, whereas `aws.us-east.us-east-1c` has one replica. YB-Load balancer always honours the value of `min_num_replicas` and have at least one replica in each cloud.region.zone, and the additional replicas may be moved around based on the cluster load. 
```

Comment:
I think here let us explain what happened as well. It feels like we immediately jump into the point about "2 additional replicas", I think maybe we should say, in the above example, the three min_num-replicas fields sum up to 3, whereas the total replication factor required is 5. And then we can go on to say that although this is a valid input and the creation goes through fine, a clarifying NOTICE appears, stating that the 2 additional replicas will be placed at the LB's discretion.
But will leave this up to you :)

Reply:
Modified the description as follows:

In the preceding example, the three `min_num_replicas` fields sum up to 3, whereas the total required replication factor is 5. As a result, even though `us_east_tablespace` has been created successfully, a notice is displayed after the execution stating that the additional two replicas are to be placed at the YB-Load balancer’s discretion in  `aws.us-east.us-east-1a`, `aws.us-east.us-east-1b`, and `aws.us-east.us-east-1c`. Based on the load, `aws.us-east.us-east-1a` may have three replicas and the other zones may have one, or it might later change such that `aws.us-east.us-east-1a` and `aws.us-east.us-east-1b` have two replicas each, whereas `aws.us-east.us-east-1c` has one replica. YB-Load balancer always honours the value of `min_num_replicas` and have at least one replica in each cloud.region.zone, and the additional replicas may be moved around based on the cluster load.